libstorage/drivers/storage/dobs/storage/dobs_storage.go

```diff
@@ -166,7 +188,6 @@ func (d *driver) VolumeInspectByName(
 	ctx types.Context,
 	volumeName string,
 	opts *types.VolumeInspectOpts) (*types.Volume, error) {
-
```

Comment:
Please don't remove this whitespace. It makes the contents of the function block have the same indentation as the line above, and hurts readability.